From c9f71eb0690a340a52a0d77bc9501588ac24333b Mon Sep 17 00:00:00 2001
From: Antoine Gerschenfeld <antoine.gerschenfeld@cea.fr>
Date: Fri, 7 Mar 2025 14:44:19 +0100
Subject: [PATCH 3/7] Add missing VTK module depends

---
 plugin/MEDReaderIO/vtk.module | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugin/MEDReaderIO/vtk.module b/plugin/MEDReaderIO/vtk.module
index f2b5567..1c68bcc 100755
--- a/plugin/MEDReaderIO/vtk.module
+++ b/plugin/MEDReaderIO/vtk.module
@@ -8,6 +8,7 @@ DEPENDS
   ParaView::VTKExtensionsIOCore
 PRIVATE_DEPENDS
   VTK::IOLegacy
+  VTK::FiltersParallelDIY2
   ParaView::VTKExtensionsFiltersRendering
   ParaView::VTKExtensionsFiltersGeneral
   ParaView::VTKExtensionsMisc
-- 
2.48.1

