From 35372adf6c4cb4b4b9ffe8b22573b59f289c3d3e Mon Sep 17 00:00:00 2001
From: Antoine Gerschenfeld <antoine.gerschenfeld@cea.fr>
Date: Fri, 7 Mar 2025 14:43:22 +0100
Subject: [PATCH 2/7] remove-custom-install

---
 plugin/ParaViewPlugin/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/plugin/ParaViewPlugin/CMakeLists.txt b/plugin/ParaViewPlugin/CMakeLists.txt
index a530d31..49e19ab 100644
--- a/plugin/ParaViewPlugin/CMakeLists.txt
+++ b/plugin/ParaViewPlugin/CMakeLists.txt
@@ -121,9 +121,3 @@ paraview_add_plugin(MEDReader
   MODULES MEDReaderIO
   MODULE_FILES "${CMAKE_CURRENT_SOURCE_DIR}/../MEDReaderIO/vtk.module"
   )
-
-install(TARGETS MEDReader
-  RUNTIME DESTINATION lib/paraview
-  LIBRARY DESTINATION lib/paraview
-  ARCHIVE DESTINATION lib/paraview
-  )
-- 
2.48.1

