From 433a5a0fc8baa11333e2f3f0fee17da9b8b8a9ce Mon Sep 17 00:00:00 2001
From: Antoine Gerschenfeld <antoine.gerschenfeld@cea.fr>
Date: Fri, 7 Mar 2025 14:45:53 +0100
Subject: [PATCH 4/5] pqTreeWidget margins API

---
 plugin/ParaViewPlugin/pqAbstractFieldsWidget.cxx | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/plugin/ParaViewPlugin/pqAbstractFieldsWidget.cxx b/plugin/ParaViewPlugin/pqAbstractFieldsWidget.cxx
index 94194e1..ebc498d 100644
--- a/plugin/ParaViewPlugin/pqAbstractFieldsWidget.cxx
+++ b/plugin/ParaViewPlugin/pqAbstractFieldsWidget.cxx
@@ -74,9 +74,8 @@ QSize pqAbstractFieldsWidget::sizeHint() const
 {
   // TreeWidget sizeHintForRow is too low, correcting to +3.
   int pix = (this->TreeWidget->sizeHintForRow(0) + 3) * this->NItems;
-  int margin[4];
-  this->TreeWidget->getContentsMargins(margin, margin + 1, margin + 2, margin + 3);
-  int h =  pix + margin[1] + margin[3];
+  QMargins margin = this->TreeWidget->contentsMargins();
+  int h =  pix + margin.top() + margin.bottom();
   if (this->visibleHeader)
     {
     h += this->TreeWidget->header()->frameSize().height();
-- 
2.48.1

