From 7f14e86ac997a8ec23ba05ccff8855033966e3e8 Mon Sep 17 00:00:00 2001
From: Antoine Gerschenfeld <antoine.gerschenfeld@cea.fr>
Date: Fri, 7 Mar 2025 14:42:38 +0100
Subject: [PATCH 1/5] Fix MEDLoaderForPV install dirs

---
 plugin/MEDLoaderForPV/CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/plugin/MEDLoaderForPV/CMakeLists.txt b/plugin/MEDLoaderForPV/CMakeLists.txt
index cae5f7b..653aa5f 100644
--- a/plugin/MEDLoaderForPV/CMakeLists.txt
+++ b/plugin/MEDLoaderForPV/CMakeLists.txt
@@ -28,5 +28,7 @@ ELSE(MEDREADER_USE_MPI)
   TARGET_LINK_LIBRARIES(MEDLoaderForPV ${MEDCoupling_medloader})
 ENDIF(MEDREADER_USE_MPI)
 
-install(TARGETS MEDLoaderForPV 
-        DESTINATION lib/salome)#${CMAKE_INSTALL_LIBDIR}
+install(TARGETS MEDLoaderForPV
+  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
-- 
2.48.1

